app/controllers/conversations_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ConversationsController < ApplicationController
+  include ApplicationHelper
+
+  def create
+    @conversation = Conversation.new(conversation_params)
+    @comment = @conversation.comment
+    @movie = @comment.movie
+    @comments = @movie.comments
+    if @conversation.save
+      activity_logger('conversation', @movie)
+      redirect_to movie_path(@movie)
+    else
+      render 'movies/show'
+    end
+  end
+
+  def edit
+    @conversation = Conversation.find_by(id: params[:id])
+  end
+
+  def destroy
+    @conversation = Conversation.find(params[:id])
+    @movie =@conversation.comment.movie
+    @conversation.destroy
+    redirect_to movie_path(@movie)
```

Comment:
Heads up: none of these instance variables should be instance variables, since none of them are being passed along. Let's not expose anything more than it needs to be.